test/core-api-v1.js

```diff
@@ -147,4 +147,8 @@ describe('Core API v1', () => {
         done();
       });
   });
+
+  it('returns a 500 if any source plugins fail');
+
+  it('returns a 429 if any source plugins have a warning');
```

Comment:
I filed the need to complete the spec to define how we'll write the code that these tests will test (whew, that was a mouthful) as issue #107.